### PR TITLE
Elexpander update

### PR DESCRIPTION
**Pull request by elexpander**
Added restaurants and activities for elexpander

### CODE REVIEW

**Review by linuxandsarcasm — COMMENTED** [review]
the lack of vowels under activities does not seem correct

**Review by elexpander — COMMENTED** [review]
jjhh

File: elexpander-shr.md [hunk 1]
```diff
@@ -1 +1,14 @@
 # Hammersmith
+
+
+### Restaurants
+asfasdf asdf
+asdf safasdf asdfasdf asd fasdfa sdfasd f
+asfasdf asdfa asdfasdf asfsa
+asfasf asdf as
```
Comment (sskhub) [inline elexpander-shr.md:8]: looks good

**Review by sskhub — COMMENTED** [review]
awesome

**Pull request merged**